README.md

```diff
@@ -60,15 +60,15 @@ algorithms for signing your credentials. For now:
 
 Where to store the private keys?
 
-TODO: Add a brief discussion of where to store the private keys. Point to
-several recommended Wallet or KMS libraries.
+It is recommended to use a [Key Management Service](https://en.wikipedia.org/wiki/Key_management) such as:
+
+- [bedrock kms](https://github.com/digitalbazaar/bedrock-kms)
+- [amazon kms](https://aws.amazon.com/kms/)
 
-Use `await keyPair.export()`
 
 #### Publishing the Public Key
 
-TODO: Explain `documentLoader` / key resolvers, and where to put the public
-key so that the verifier can get to it.
+Public Keys can be stored in your [Key Management Service](https://en.wikipedia.org/wiki/Key_management), as static data on api end points, or even recreated from seeds. Regardless of which method that you use, your public key must be deferenced by a keyResolver so the verifier can use it to verify your credential. Your credential itself must be referenced by documentLoader.
```

Comment:
Do KMSs manage public keys as well? I thought they were just for private keys.

Reply:
@dmitrizagidulin I don't actually know. I believe that bedrock-KMS does actually handle public key material too.

Reply:
@dlongley @davidlehn  can we resolve this lingering issue and get the the expanded read me in?

Reply:
@dmitrizagidulin this was removed so I'm marking this as resolved.